.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "files.associations": {
+        "*.h": "c",
+        "*.c": "c",
+        "iosfwd": "c"
+    }
+}
```

Comment:
get this out of here, lol.

We can add `.vscode` to the .gitignore, I added it to my system one, but we did add it to some of our repos.

Reply:
added to gitignore. and removed the file